CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -715,10 +717,6 @@ \subsubsection*{Scope and Challenges}
 \item
   Revisiting digitisation algorithms to improve performance and
   exploring opportunities for code sharing among experiments.
-\item
-  Recruiting, training, retaining human resources in all areas of
-  expertise pertaining to the simulation domain, including software and
-  physics.
```

Comment:
Just in case, as I did not check: this item was meant to be removed?

Reply:
Yes, one of the referees said it was out of place and we decided to to only discuss this aspect in Ch4.